docs/integrations/common-workflow-steps/build-cache/cache-push.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Cache Push 
+description: Optimize your workflow by utilizing Cache Push to save and streamline data access, improving speed and reliability in your projects.
+tags: [cache push, optimization, storage, dependencies, cache structure]
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Cache Push
+
+Every single build at Appcircle runs in a clean state. It means that all files and folders that are not versioned in the Git repository are lost when the build pipeline is completed. For example, install dependencies or build artifacts. If you need to keep those files and folders, you can use the Appcircle **Cache Push** and [**Cache Pull**](./cache-pull) components.
+
+With cache, you can persist any resource that is ignored by Git. So you can transfer files and folders between build pipelines. Sometimes it may speed up your build, or it may help if you have reliability issues with the original download location for dependencies. But keep in mind that the cache is uploaded to or downloaded from a remote location. It may help you in some cases, but **it's not a guaranteed way to speed up builds**. You should try and see the actual results of your project.
+
+The cache is stored as a single archive file. **Cache Push** and [**Cache Pull**](./cache-pull) components work in coordination on the same cache file defined with a label. Cache labeling helps you organize your caches. With custom labeling, you can have different chunks of caches, and you can share some caches between branches or build profiles. For further information, please check out the following documentation:
+- [How to Share Files Between Pipelines](./how-to-share-file-between-pipelines)
+- [How to Share Files Between Build Profiles](./how-to-share-file-between-build-profiles)  
+
+When you drag and drop the **Cache Push** component into your [workflow](../../../workflows/why-to-use-workflows), it comes with pre-defined values according to your project type. For example, in the case of Android projects, it comes with pre-defined [Gradle cache](https://docs.gradle.org/current/userguide/build_cache.html) paths, which should prove useful for most Android apps.
```

Comment:
Links should be the exact path.

Wrong:
`(./cache-push)`

Correct:
`(/workflows/common-workflow-steps/cache-push)`